Snowflake.Data/Core/FileTransfer/SFFileTransferAgent.cs

```diff
@@ -666,6 +668,26 @@ private void initEncryptionMaterial()
             }
         }
 
+        private int GetFileTransferMaxBytesInMemory()
+        {
+            if (!Session.properties.TryGetValue(SFSessionProperty.FILE_TRANSFER_MAX_BYTES_IN_MEMORY, out var maxBytesInMemoryString))
+            {
+                return FileTransferConfiguration.DefaultMaxBytesInMemory;
+            }
+            if (string.IsNullOrEmpty(maxBytesInMemoryString))
+            {
+                return FileTransferConfiguration.DefaultMaxBytesInMemory;
+            }
+            try
+            {
+                return int.Parse(maxBytesInMemoryString);
+            }
+            catch (Exception e)
+            {
```

Comment:
For the driver user I would log at least maxBytesInMemory string to indicate why we switch to default, what do you think?

Reply:
actually it is impossible, because we have a validation that if it impossible to parse, establishing connection would fail. So it is just in case, but it won't happen.